### PR TITLE
Initial draft: basic build instructions on Readme

### PR DESCRIPTION
**Pull request by rjan90**
Inital draft for basic build instructions on the Readme page. Issue: #6348

### CODE REVIEW

**Comment by jennijuju** [comment]
Thank you @rjan90 for working on this!!

File: README.md [hunk 1]
```diff
@@ -50,6 +50,112 @@ When implementing a change:
...
+
+Lotus needs [rustup](https://rustup.rs). The easiest way to install it is:
+
+```bash
+wget -c https://golang.org/dl/go1.16.2.linux-amd64.tar.gz -O - | sudo tar -xz -C /usr/local
+```
+
+#### Go
+
+To build Lotus, you need a working installation of [Go 1.16.1 or higher](https://golang.org/dl/):
```
Comment (jennijuju) [inline README.md:80]:
```suggestion
To build Lotus, you need a working installation of [Go 1.16.4 or higher](https://golang.org/dl/):
```

File: README.md [hunk 2]
```diff
@@ -50,6 +50,112 @@ When implementing a change:
...
+
+Once all the dependencies are installed, you can build and install the Lotus suite (`lotus`, `lotus-miner`, and `lotus-worker`).
+
+1. Clone the repository:
+
+   ```sh
+   git clone https://github.com/filecoin-project/lotus.git
+   cd lotus/
+   ```
+
```
Comment (jennijuju) [inline README.md:105]:
```suggestion
> Note: The default branch `master` is the dev branch where the latest new features, bug fixes and improvement are in. However, if you want to run lotus on Filecoin mainnet and want to run a production-ready lotus, get the latest release[ here](https://github.com/filecoin-project/lotus/releases).
```

File: README.md [hunk 3]
```diff
@@ -50,6 +50,112 @@ When implementing a change:
...
+   export CGO_CFLAGS_ALLOW="-D__BLST_PORTABLE__"
+   export CGO_CFLAGS="-D__BLST_PORTABLE__"
+   ```
+
+   This is due to a Lotus bug that prevents Lotus from running on a processor without `adx` instruction support, and should be fixed soon.
+
+5. Build and install Lotus:
+
+   ```sh
+   make clean all
```
Comment (jennijuju) [inline README.md:144]:
```suggestion
   make clean all #mainnet
```

**Review by jennijuju — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -50,6 +50,112 @@ When implementing a change:
...
+   Currently, the latest code on the _master_ branch corresponds to mainnet.
+
+3. If you are in China, see "[Lotus: tips when running in China](https://docs.filecoin.io/get-started/lotus/tips-running-in-china/)".
+4. Depending on your CPU model, you will want to export additional environment variables:
+
+   If you have **an AMD Zen or Intel Ice Lake CPU (or later)**, enable the use of SHA extensions by adding these two environment variables:
+
+   ```sh
+   export RUSTFLAGS="-C target-cpu=native -g"
+   export FFI_BUILD_FROM_SOURCE=1
```
Comment (magik6k) [inline README.md:127]: I'd say that the basic instructions should probably use the prebuilt proof binaries - this way you don't have to install rustup, and build the proofs library which takes a while. Then we can link to the instructions on docs.filecoin.io for people who may want a full from-source build

Comment (jennijuju) [reply README.md:127]: I don’t think the proof prebuilds are compatible with different systems atm? And isn’t building from source a pretty standard thing to do now?

Comment (jennijuju) [reply README.md:127]: oh checked with nemo, i was wrong, magik is right. @rjan90 mind making the update?

**Review by magik6k — APPROVED** [review]
This is great, thanks for the PR!

**Pull request merged**